infrastructure/code/template/details.html

```diff
@@ -451,84 +713,169 @@ <h2 class="severity-title">{{.IssueTitle}}</h2>
         <!-- Start Fix Analysis section for tab id="fix-analysis-tab" -->
         <div data-content="fix-analysis" id="fix-analysis-content"
           class="tab-content main-tab-content {{if not .IsIgnored}}is-selected{{end}}">
-
-          {{if .HasAIFix }}
-          <!-- AI Fix -->
-          <section class="ai-fix">
-            <div class="fix-analysis-description">Unsanitized input from cookies flows into send, where it is used to
-              render an HTML page returned to the
-              user. This may result in a Cross-Site Scripting attack (XSS).</div>
-            <h2 class="fix-analysis--header">
-              DeepCode AI Fixes
-            </h2>
-            <span>⚡️ Fix this issue by generating a solution using Snyk DeepCode AI</span>
-            <button>✨ Generate fix using Snyk DeepCode AI</button>
-          </section>
-          {{else}}
-          <!-- We need Data Flow and Fixed Code Examples -->
-          <!-- Data Flow -->
-          <section class="data-flow-section">
-            <h2 class="data-flow-header">
-              Data Flow - {{len .DataFlow}} {{if gt (len .DataFlow) 1}}steps{{else}}step{{end}}
-            </h2>
-            <div class="data-flow-body">
-              <table class="data-flow-body">
-                <tbody>
-                  {{range .DataFlowTable}}
-                  <tr class="data-flow-row">
-                    <td class="data-flow-number">{{.Number}}</td>
-                    <td class="data-flow-clickable-row" file-path="{{.FilePath}}" start-line="{{.StartLine}}"
-                      end-line="{{.EndLine}}" start-character="{{.StartCharacter}}" end-character="{{.EndCharacter}}">
-                      {{.FileName}}:{{.StartLineValue}}
-                    </td>
-                    <td class="data-flow-delimiter">|</td>
-                    <td class="data-flow-text">{{.Content}}</td>
-                  </tr>
-                  {{end}}
-                </tbody>
-              </table>
+          <section id="suggestion-info" class="delimiter-top">
```

Comment:
Instead of deciding what to render when generating the template, we use classes to decide whether to hide panels if the issue has an AI fix. This way, in Intellij we can remove/add the `hidden` class such that we always hide AI fixes

Reply:
Yeah! That makes a lot of sense 🙌  There are a few [downsides](https://stackoverflow.com/a/5448974) when using CSS rules to hide components, but I thing for our context that is the IDE's land, we don't need to care about:

- **Initial Load**: All hidden elements are still sent to the IDE, which can increase the initial load time and the size of the HTML document. 
- **Rendering Overhead**: The IDE still needs to parse and render these hidden elements, which can add some overhead and potentially slow down the IDE, especially in larger projects.
- **Data Exposure**: For our case, I guess it's not a big problem but all the hidden elements are still accessible from the DevTool meaning that with just removing the class `hidden` the element is exposed.

Reply:
Yep in this case, since it's temporary situation, this is okay 😄